parl/layers/layer_wrappers.py

```diff
@@ -91,15 +92,15 @@ def create_new_para_name(attr):
 
     @property
     def param_name(self):
-        if self.param_attr:
-            return self.param_attr.name
+        if self.attr_holder.param_attr:
+            return self.attr_holder.param_attr.name
         else:
             return None
 
     @property
     def bias_name(self):
-        if self.bias_attr:
-            return self.bias_attr.name
+        if self.attr_holder.bias_attr:
+            return self.attr_holder.bias_attr.name
         else:
             return None
 
```

Comment:
Except property of `param_name` and `bias_name` above, users can't acquire other parameter_names, how about adding a new property for users to get all the parameters_name ?

Reply:
done